docs/use-cases/navigate.md

```diff
@@ -1,8 +1,54 @@
 ---
-title: "Teleoperate and navigate rovers"
+title: "Teleoperate and navigate with rovers"
 linkTitle: "Teleoperate and navigate with rovers"
-weight: 60
+weight: 50
 type: "docs"
-layout: "empty"
-canonical: "tutorials/services/navigate-with-rover-base/"
+tags: ["navigation", "motion", "services", "base", "base remote control"]
+no_list: true
+description: "Use Viam to teleoperate and navigate rover bases."
+image: "/services/icons/navigation.svg"
+imageAlt: "Navigation"
+images: ["/services/icons/navigation.svg"]
 ---
+
+If you have a rover base, you can teleoperate and navigate it with Viam.
+Remotely control your base after configuring your machine on the app's **Control** tab, and set up autonomous navigation with the [navigation service](/mobility/navigation/).
+
+<table>
+  <tr>
+    <th>{{<imgproc src="/use-cases/base-control.png" class="fill alignright" resize="200x" style="max-width: 200px" declaredimensions=true alt="Base control card">}}
+      <b>1. Teleoperate</b>
+      <p>Remotely control your rover base through the <a href="https://app.viam.com">Viam app.</a> Create an account and add a machine, <a href="/get-started/installation/">install viam-server</a>, and <a href="/components/base/">configure your base</a>. Then, go to the <b>Control</b> tab and access a remote control card for your base, with an interface for controlling speed, direction, and power.</p>
+    </th>
+  </tr>
+  <tr>
+    <th>{{<imgproc src="/services/icons/base-rc.svg" class="fill alignleft" resize="200x" style="max-width: 200px" declaredimensions=true alt="Base remote control service icon.">}}
+      <b>2. Program to move</b>
+      <p>Remotely control your rover base programmatically with a <a href="/sdks/">Viam SDK.</a> Make calls to the <a href="/mobility/motion/">motion service.</a>
```

Comment:
```suggestion
      <p>Remotely control your rover base programmatically with the <a href="/components/base/#api">base API</a>.
```
Take period out of link.

Motion service would be hard for most users to use effectively on a base without movement sensors and nav; let's take that out and focus on the path most users should take.

What do you think of linking directly to the base API so they can quickly see what they have available? Could see the argument for leaving as the SDKs page since that shows more different languages than the base API page currently does, but I think it'd help folks.

Reply:
should the motion service then be mentioned with steps 3 and 4?